### ISSUE TITLE
Listing tips by topic

### ISSUE THREAD

**Issue by Beliavsky** [zmoon/FortranTipBrowser#5]
I categorized the tips by topic at https://github.com/Beliavsky/FortranTip/blob/main/topics.md . Where to place a tip is ad-hoc, since a tip may be at an intersection of topics. It would be nice if the TipBrowser could show tips by topic. In another issue I see that tagging tips was mentioned.

**Comment by zmoon** [comment]
Yes, I saw the Tweet today, very nice. I am planning to implement the topic lists as sections on the tips index page (<https://zmoon.github.io/FortranTipBrowser/tips/>).

> Where to place a tip is ad-hoc, since a tip may be at an intersection of topics.

To me it would be ok, if one applies to multiple topics, that you place it in multiple. I wasn't sure if you had done that or not.

**Comment by zmoon** [comment]
> In another issue I see that tagging tips was mentioned.

This was in #1. Maybe I will take this chance to try ablog

**Comment by Beliavsky** [comment]
> To me it would be ok, if one applies to multiple topics, that you place it in multiple. I wasn't sure if you had done that or not.

In my list by topic each tip appears only once, because I don't want a reader who goes through the tweets in order to see repetition. Of course you can do it differently on FortranTipBrowser.
